haskell-mode.el

```diff
@@ -10,6 +10,7 @@
 ;;          2003      Dave Love <fx@gnu.org>
 ;; Keywords: faces files Haskell
 ;; URL: https://github.com/haskell/haskell-mode
+;; Version: 13.07
```

Comment:
I need a version here otherwise cask is not happy.
What should be the right one here?